src/component/app/app.component.jsx

```diff
@@ -1,36 +1,35 @@
-import React from 'react'
-import { Provider } from 'react-redux'
-import { ThemeProvider } from 'styled-components'
-import theme from '../../config/theme'
-import * as S from './app.style'
+import React from "react";
+import { Provider } from "react-redux";
+import { ThemeProvider } from "styled-components";
+import theme from "../../config/theme";
+import * as S from "./app.style";
 //Provider it's actually a react component
 //It's bassicaly the glue for react and redux
 
-import store from '../../store'
+import store from "../../store";
 //The store holds the state
-import { Menu } from '../menu'
-import { Home } from '../home'
-import { Auctions } from '../auctions'
-import { Sell } from '../sell'
-import { AuctionPage } from '../auction-page'
-import { Footer } from '../footer'
+import { Menu } from "../menu";
+import { Home } from "../home";
+import { Auctions } from "../auctions";
+import { Sell } from "../sell";
+import { AuctionPage } from "../auction-page";
+import { Footer } from "../footer";
 
 const App = () => {
-
-  return(
+  return (
     <ThemeProvider theme={theme}>
       <Provider store={store}>
         <Menu />
         <S.StyledRouter>
           <Home path="/" />
           <Auctions path="auctions" />
           <Sell path="sell" />
-          <AuctionPage path="auctionPage" />
+          <AuctionPage path="/auction/:id" />
         </S.StyledRouter>
         <Footer />
       </Provider>
     </ThemeProvider>
-  )
-}
+  );
+};
 
-export { App }
+export { App };
```

Comment:
Any reason we are exporting like this?
If you want a named export, just use the keyword 'export' when declaring the App component (line 18).
If you want a default export, use 'export default App' here.

Your current implementation exports a new Object that has App as property. No reason to declare this new Object.